library/core/src/iter/range.rs

```diff
@@ -1224,15 +1224,15 @@ impl<T: TrustedStep> RangeInclusiveIteratorImpl for ops::RangeInclusive<T> {
 
         while self.start < self.end {
             // SAFETY: just checked precondition
-            let n = unsafe { Step::backward_unchecked(self.end.clone(), 1) };
+            let n = unsafe { Step::backward_unchecked(self.end().clone(), 1) };
```

Comment:
What happened here? Shouldn't this be:
```suggestion
            let n = unsafe { Step::backward_unchecked(self.end, 1) };
```